.changeset/tame-falcons-clap.md

```diff
@@ -0,0 +1,5 @@
+---
+'@web3modal/common': minor
+---
+
+refactor(common): utils
```

Comment:
@Simon-He95 we don't create changeset files unless we release. Could you remove it please?

Reply:
sure